StreamChatClient_v3/Controllers/ChannelController.swift

```diff
@@ -119,6 +119,7 @@ public class ChannelControllerGeneric<ExtraData: ExtraDataTypes>: Controller, De
         return observer
     }()
     
+    private var eventObservers = [EventObserver]()
```

Comment:
nit: we usually use the following syntax for this:
```swift
private var eventObservers: [EventObserver] = []
```

Reply:
fixed in 791b4a3673a2127949ad673e06f1ae90366330f9